compatibility-verifier/compCheck.sh

```diff
@@ -169,6 +183,11 @@ COMPAT_TESTER_PATH="pinot-integration-tests/target/pinot-integration-tests-pkg/b
 # create subdirectories for given commits
 workingDir=$1
 testSuiteDir=$(absPath "$2")
+
+brokerConfigFile=${testSuiteDir}/BrokerConfig.conf
```

Comment:
```suggestion
brokerConfigFile=${testSuiteDir}/config/BrokerConfig.conf
```
Same for other files also. We want to have them all in the config/ directory

Reply:
done!